ports/unix/mbedtls/mbedtls_config_port.h

```diff
@@ -32,6 +32,9 @@
 // Enable mbedtls modules
 #define MBEDTLS_TIMING_C
 
+// Enable DTLS
+#define MBEDTLS_SSL_PROTO_DTLS
```

Comment:
To enable this feature on all ports that use mbedtls, this can be moved to `extmod/mbedtls/mbedtls_config_common.h`.

Reply:
Moved to the common location!